Makefile

```diff
@@ -90,6 +90,18 @@ build: ## Generate a binary in ./bin
 	@echo "==> Building binary"
 	go build -ldflags "${LINKER_FLAGS}" -o ${DESTINATION}
 
+.PHONY: e2e-test-atlas
+e2e-test-atlas: build ## Run Atlas E2E tests
+	@echo "==> Running E2E tests..."
+	# the target assumes the MCLI-* environment variables are exported
+	go test -v -p 1 -parallel 1 -tags=e2e ./e2e/atlas...
```

Comment:
This is merely informative and you don't need to change anything here but another thing you could've done here is to change the build tags to something like`-tags=atlas_e2e` 
and then in the test file change line 14 to `+build atlas_e2e`

This is a golang thing to specify build targets
By default go will run all your test files, and that tag is how we prevent e2e from running when you do a simple `go test .` command